script/tool/test/update_excerpts_command_test.dart

```diff
@@ -33,12 +33,13 @@ void runAllTests(MockPlatform platform) {
       ));
   });
 
-  Future<void> testInjection(String before, String source, String after,
+  Future<void> testInjection(
+      String before, String source, String after, String fileName,
```

Comment:
Nit throughout the PR: "filename" is generally a single word in programming, so the variable would just be `filename`.